example/app.tsx

```diff
@@ -5,17 +5,26 @@ import {Item, AppWrapper, AppHeader, AppFooter} from './styled';
 
 export default class App extends Component {
   render() {
-    const items = Array(20).fill().map((value, key) => 
-      <Item key={key}>{key}</Item>
-    );
+    const items = [];
```

Comment:
I was testing it in IE11. Flex doesn't work there, but I couldn't see a thing because of this JS.

Reply:
Good call, I wasn't bothering about having IE support, but if you fixed it awesome!

Reply:
It does work in Edge, Chrome, Safari, Firefox. IE11 doesn't support Flex, but it shows content, which is good.